.ci/pytorch/windows/arm64/bootstrap_buildtools.bat

```diff
@@ -18,31 +19,46 @@ curl -L -o "%INSTALLER_FILE%" %DOWNLOAD_URL%
-    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+    "%INSTALLER_FILE%" --norestart --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+        --add Microsoft.VisualStudio.Component.Roslyn.Compiler ^
+        --add Microsoft.Component.MSBuild ^
+        --add Microsoft.VisualStudio.Component.CoreBuildTools ^
+        --add Microsoft.VisualStudio.Workload.MSBuildTools ^
+        --add Microsoft.VisualStudio.Component.Windows10SDK ^
+        --add Microsoft.VisualStudio.Component.VC.CoreBuildTools ^
+        --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
+        --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest ^
         --add Microsoft.VisualStudio.Component.Windows11SDK.22621 ^
-        --add Microsoft.VisualStudio.Component.VC.ASAN ^
         --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
-        --add Microsoft.VisualStudio.Component.VC.Tools.ARM64 ^
-        --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+        --add Microsoft.VisualStudio.Component.TestTools.BuildTools ^
+        --add Microsoft.VisualStudio.Component.VC.ASAN ^
+        --add Microsoft.VisualStudio.Component.TextTemplating ^
+        --add Microsoft.VisualStudio.Component.VC.CoreIde ^
+        --add Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Core ^
+        --add Microsoft.VisualStudio.Workload.VCTools ^
+        --add Microsoft.VisualStudio.Component.VC.Tools.ARM64EC ^
+        --add Microsoft.VisualStudio.Component.VC.Tools.ARM64
 ) else if "%MSVC_VERSION%" == "14.40" (
-    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --fix --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
         --add Microsoft.VisualStudio.Component.Windows11SDK.22621 ^
         --add Microsoft.VisualStudio.Component.VC.ASAN ^
         --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
         --add Microsoft.VisualStudio.Component.VC.14.40.17.10.ARM64 ^
         --add Microsoft.VisualStudio.Component.VC.14.40.17.10.x86.x64
 ) else if "%MSVC_VERSION%" == "14.36" (
-    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --fix --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
         --add Microsoft.VisualStudio.Component.Windows11SDK.22621 ^
         --add Microsoft.VisualStudio.Component.VC.ASAN ^
         --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
         --add Microsoft.VisualStudio.Component.VC.14.36.17.6.ARM64 ^
         --add Microsoft.VisualStudio.Component.VC.14.36.17.6.x86.x64
 )
 
+echo exitcode = %errorlevel%
+
 :: Check if installation was successful
 if %errorlevel% neq 0 (
-    echo "Failed to install Visual Studio Build Tools with C++ components. (exitcode = %errorlevel%)"
```

Comment:
This and the change of the previous lines does not make much sense: When the `%errorlevel%` is non-zero it's better to print the value together with the error message, when it's zero it's kind of unecessary to print it.

Reply:
OK, thank you for the explanation.